### DISCUSSION TITLE
Is it possible to use greek letters in the acronym package from latex in Quarto ?

### DISCUSSION THREAD

**Discussion by Minh-AnhHuynh** [quarto-dev/quarto-cli#6247] [Q&A] [answered]
### Description

Hello,

I am facing the problem of not having the greek letters appear in the acronyms from using the acronym latex package in Quarto for pdf output.

I have written about this problem here and it seems the issue is not reproducible with pure latex code, hence it seems Quarto is doing something to the acronym package.
https://tex.stackexchange.com/questions/689291/how-to-use-acronym-package-to-use-greek-letterswhile-using-unicode-greek-letters

Here is my Quarto code.

```
---
title: "quarto-test-env"
format: pdf
header-includes:
- \usepackage{acronym}
- \usepackage{upgreek}
keep-tex: true
mainfont: "Times New Roman"
---

# Acronyms


\begin{acronym}
  \acro{ifng}[IFN-$\mupgamma$]{interferon gamma}
  \acro{tnfa}[TNF-$\mupalpha$]{tumor necrosis factor alpha}
  \acro{nfkb}[NF-κB]{nuclear factor-kappa B}
\end{acronym}

# Regular text

- \ac{tnfa}
- \ac{ifng}
- \ac{nfkb}

I want TNF-α and not \ac{tnfa} ; IFN-γ and not \ac{ifng}.
When I use unicode characters in acronym such as \ac{nfkb}, it doesn't display in the acronym but in the regular text only.
```

**Comment by cderv** [comment]
It seems to come from `scrartcl` `documentclass`. Quarto uses Koma Script by default (https://quarto.org/docs/output-formats/pdf-basics.html#document-class). If you use `article` I think you won't see the issue. 

```yaml
documentclass: article
```

> the issue is not reproducible with pure latex cod

What is the documentclass you set in LaTeX ? If you try with `\documentclass[letterpaper]{scrartcl}`, I believe you could reproduce the issue. 

So something in Koma Script does conflict with acronym. Does not seem to be a quarto issue.

**Reply by Minh-AnhHuynh** [reply]
Using `\documentclass[letterpaper]{scrartcl}` with `\setmainfont{Times New Roman}` I end up with an undefined control sequence (using acronym). It works if I remove any mention of the acronym package. Everything seems to work once I use the `libertine `font somehow.

I do use `scrreprt `as the document class to use `listof=totoc` as a document option, making use of the default documentclass. It doesn't seem to work either if I use the `article` class.

For the record here is the code I used. I only changed the document class.

```
\documentclass{article}
\usepackage{acronym}
\setmainfont{Times New Roman}

\begin{document}
\begin{acronym}
  \acro{ifng}[IFN-γ]{interferon gamma}
  \acro{tnfa}[TNF-α]{tumor necrosis factor alpha}
\end{acronym}

\ac{tnfa}
\ac{ifng}
\end{document}

```

**Reply by cderv** [reply, marked as answer]
What I meant is that  this does seem to work for me by adding `documentclass: article`
````markdown
---
title: "quarto-test-env"
format: 
  pdf:
    documentclass: article
header-includes:
- \usepackage{acronym}
- \usepackage{upgreek}
mainfont: "Times New Roman"
keep-tex: true
---

# Acronyms

\begin{acronym}
  \acro{ifng}[IFN-$\mupgamma$]{interferon gamma}
  \acro{tnfa}[TNF-$\mupalpha$]{tumor necrosis factor alpha}
  \acro{nfkb}[NF-κB]{nuclear factor-kappa B}
\end{acronym}

# Regular text

- \ac{tnfa}
- \ac{ifng}
- \ac{nfkb}

I want TNF-α and not \ac{tnfa} ; IFN-γ and not \ac{ifng}.
When I use unicode characters in acronym such as \ac{nfkb}, it doesn't display in the acronym but in the regular text only.

````

![image](https://github.com/quarto-dev/quarto-cli/assets/6791940/ce126456-d129-4b73-9df8-6a7c92ae1402)

Does it work for you ?

**Reply by Minh-AnhHuynh** [reply]
Oh I misread, I apologize. Yes, this seems to resolve the issue! Thank you very much.
